library/src/webapp/editor/ckextraplugins/oxam/js/oxam-embed.js

```diff
@@ -36,8 +36,10 @@ $.fn.oxamEmbed = function(options) {
     var params = {};
 
     for (i in tmpParams) {
-      var param = tmpParams[i].split('=');
-      params[param[0]] = param[1];
+      if (i!='contains'){
```

Comment:
The better fix is normally to switch to a sequential loop: http://stackoverflow.com/questions/3010840/loop-through-array-in-javascript